pkg/network/state.go

```diff
@@ -1100,22 +1123,112 @@ type aggregateConnection struct {
 	count             uint32
 }
 
+type aggregationKey struct {
+	string
```

Comment:
is there a reason to embed string and not use `connKey string`?

Reply:
yeah, to me something like `connKey` and a comment saying that this is populated with data from `*ConnectionStats.ByteKey()` would make the code easier to follow